packages/builders/src/components/ActionRow.ts

```diff
@@ -99,37 +70,122 @@ export class ActionRowBuilder<ComponentType extends AnyComponentBuilder> extends
 	 * ```
 	 */
 	public constructor({ components, ...data }: Partial<APIActionRowComponent<APIActionRowComponentTypes>> = {}) {
-		super({ type: ComponentType.ActionRow, ...data });
-		this.components = (components?.map((component) => createComponentBuilder(component)) ?? []) as ComponentType[];
+		super();
+		this.data = {
+			...structuredClone(data),
+			type: ComponentType.ActionRow,
+			components: components?.map((component) => createComponentBuilder(component)),
+		};
 	}
 
 	/**
-	 * Adds components to this action row.
+	 * Adds a button to this action row.
```

Comment:
This allows adding more than one. Applies to the rest of the methods in this builder

```suggestion
	 * Adds button components to this action row.
```

```suggestion
	 * Adds buttons to this action row.
```

Reply:
> Applies to the rest of the methods in this builder

No.. it doesn't...?